packages/react-magma-dom/src/components/Transition/Transition.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
This feels a bit empty. Nothing really being tested here.

Reply:
It's actually just the generated tests from plop 😂 I don't know what an actual good test would be 🤔 we could add a check for the css properties for all of them... but wouldn't snapshots already have that same exact case covered...